### PR TITLE
Add testimonial from Graphviz's use

### PR DESCRIPTION
**Pull request by mhansen**
Added some benchmarks too

### CODE REVIEW

**Comment by mhansen** [comment]
Please feel free to rewrite this or suggest some changes or condense this; it's a bit long perhaps.

File: README.md [hunk 1]
```diff
@@ -140,6 +140,27 @@ and `--github-actions` feature.
...
+>   links within both `<a>` tags and HTML redirects, and a further 62 broken
+>   anchor-links with `--check-anchors`.
+> * Hyperlink's design decision to crawl only static files (avoiding HTTP),
+>   avoids test flakiness from network requests, allowing me to confidently
+>   block merging if Hyperlink reports an error.
+>
+> In conclusion, Hyperlink fills the "static site continuous testing" niche
+> really nicely.
+
+-- Mark Hansen, Graphviz documentation maintainer:
```
Comment (untitaker) [inline README.md:162]:
```suggestion
-- Mark Hansen, Graphviz documentation maintainer
```

**Comment by untitaker** [comment]
>Please feel free to rewrite this or suggest some changes or condense this; it's a bit long perhaps.

I'll probably have to start thinking about this once the README gets unbearably long but right now it's okay, i think

**Comment by untitaker** [comment]
>Added some benchmarks too

i don't think you added any

**Comment by mhansen** [comment]
> Added some benchmarks too

Sorry I suppose they're not really benchmarks, just benchmark results: I quoted some numbers from me running 

```
$ hyperfine -i "~/Downloads/hyperlink-mac-x86_64 public/"
$ hyperfine -i "~/Downloads/hyperlink-mac-x86_64 public/ --check-anchors"
```

on the Graphviz repo.

**Comment by untitaker** [comment]
ah okay, yeah that's totally fine

**Comment by untitaker** [comment]
thanks :pray:

**Pull request merged**